src/users/collection/getItemsInFolderForUser.test.js

```diff
@@ -45,4 +50,18 @@ describe('Users - Collection - getItemsInFolderForUser', () => {
     sort.should.be.equal(by)
     sort_order.should.be.equal(order)
   })
+  it('should accept pagination', async () => {
+    const page = 1
+    const perPage = 3
+    const data = await client.getInventory({
```

Comment:
You didn't change the function name when copying-pasting, so this test does not affect `getItemsInFolderForUser`